composer.json

```
@@ -13,20 +13,20 @@
 	],
 	"require": {
 		"php": ">=7.1",
-		"composer/installers": "^1"
+		"composer/installers": "^1.12.0"
```

The WordPress plugin installers are available in `1.0.0`, so why is `1.12.0` a minimum requirement?

I'd rather see us test and add support for `^2`:

```suggestion
		"composer/installers": "^1||^2"
```

---

Good point, let me try this out.